internal/cache/bigcache/cache.go

```diff
@@ -0,0 +1,56 @@
+package bigcache
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/caos/logging"
+
+	"github.com/caos/zitadel/internal/errors"
+
+	a_cache "github.com/allegro/bigcache"
+)
+
+type Bigcache struct {
+	cache *a_cache.BigCache
+}
+
+func NewBigcache(c *Config) (*Bigcache, error) {
+	cacheConfig := a_cache.DefaultConfig(c.CacheLifetime)
+	cacheConfig.HardMaxCacheSize = c.MaxCacheSizeInMB
+	if c.CacheLifetime > 0 {
+		cacheConfig.CleanWindow = 1 * time.Minute
```

Comment:
do we want to set  the clean window to 1 minute if a `CacheLifetime` is defined?

Reply:
we could ad it to config if needed
as we use fastcache at the moment, it's not that important anyhow

Reply:
in my opinion its very bad practice to push wrong code, doesn't matter if needed or not.

Reply:
From [the readme](https://github.com/allegro/bigcache#lifewindow--cleanwindow).

`LifeWindow` is a time. After that time, an entry can be called dead but not deleted.

`CleanWindow` is a time. After that time, all the dead entries will be deleted, but not the entries that still have life.

If we want to use this feature we should set both config-fields one of both doesn't make sense.

Reply:
i removed it